honeybee_schema/radiance/properties.py

```diff
@@ -0,0 +1,173 @@
+"""Properties Schema"""
+from pydantic import Field, constr, validator, root_validator
+from typing import List, Union
+
+from .._base import IDdBaseModel
+from .modifier import Void, ModifierBase
+from .modifierset import ModifierSet
+
+
+class _PropertiesBaseAbridged(IDdBaseModel):
+    """Base class of Abridged Radiance Properties."""
+
+    modifier: str = Field(
+        default=None,
+        min_length=1,
+        max_length=100,
+        description='A string for a Honeybee Radiance Modifier.'
+        )
+
+    modifier_blk: str = Field(
+        default=None,
+        min_length=1,
+        max_length=100,
```

Comment:
Same goes for here. Just remove the length restrictions for now and do the character enforcing on the Modifier classes.